lightning/config/global.go

```diff
@@ -180,6 +180,13 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 	}
 	if *logFilePath != "" {
 		cfg.App.Config.File = *logFilePath
+	} else {
+		configFile, err := log.AdjustLogFilePath()
+		if err != nil {
+			return nil, errors.Annotate(err, "Create debug logs directory failed")
+		}
+		cfg.App.Config.File = configFile
+		fmt.Fprintf(os.Stdout, "Verbose debug logs will be written to %s.\n\n", configFile)
```

Comment:
there should still be a way to explicitly use `stderr` as log output.

i suggest using BR's approach of making the default value of the CLI parameter to be the log path.